stats/servicedstatsreporter.go

```diff
@@ -205,7 +212,10 @@ func (sr *ServicedStatsReporter) updateStats() {
 			previousTotalCPU, found := sr.previousStats[key]["totalCPU"]
 			if found {
 				if totalCPU <= previousTotalCPU {
-					plog.Warningf("Change in total CPU usage was nonpositive, skipping CPU stats update.")
+					plog.WithFields(logrus.Fields{
+						"totalCPU":         totalCPU,
+						"previousTotalCPU": previousTotalCPU,
+					}).Warn("Change in total CPU usage was nonpositive, skipping CPU stats update")
```

Comment:
This should be debug level, almost certainly. Rule of thumb: 1) Crit/Error means somebody needs to be paged 2) Warning means somebody works on fixing it that week 3) Info is human-readable positive system updates 4) Debug is anything repeating that doesn't present a loss in functionality